dataland-user-service/src/main/kotlin/org/dataland/datalanduserservice/service/MessageQueuePublisherService.kt

```diff
@@ -6,16 +6,23 @@ import org.dataland.datalandmessagequeueutils.constants.ExchangeName
 import org.dataland.datalandmessagequeueutils.constants.MessageType
 import org.dataland.datalandmessagequeueutils.constants.RoutingKeyNames
 import org.dataland.datalandmessagequeueutils.messages.data.PortfolioUpdatePayload
+import org.dataland.datalandmessagequeueutils.messages.email.EmailMessage
+import org.dataland.datalandmessagequeueutils.messages.email.EmailRecipient
+import org.dataland.datalandmessagequeueutils.messages.email.InternalEmailContentTable
+import org.dataland.datalandmessagequeueutils.messages.email.Value
+import org.dataland.datalanduserservice.model.SupportRequestData
 import org.dataland.keycloakAdapter.auth.DatalandAuthentication
+import org.dataland.keycloakAdapter.auth.DatalandJwtAuthentication
 import org.slf4j.LoggerFactory
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Service
+import java.util.UUID
 
 /**
  * Publisher Service for Portfolio Updates (Companies and Monitoring status).
```

Comment:
```suggestion
 * Publisher Service for Portfolio Updates (Companies and Monitoring status).
```
does this need to be modified?

Reply:
It does not, and locally IntelliJ does not show me any diff with main here